package.json

```diff
@@ -132,12 +135,23 @@
 		},
 		"verbose": true
 	},
+	"lint-staged": {
+		"*.js": [
```

Comment:
Why only `.js` files. We should be using Prettier for `.css`, `.less`, `.scss`, or `.json` files as well. Since, ESLint can't.

Thoughts?

Reply:
We're only discussed using Prettier for JavaScript files at this stage, I plan on proposing we also use it for SCSS & CSS files but there's some work to do here to bring [stylelint-config-wordpress](https://github.com/WordPress-Coding-Standards/stylelint-config-wordpress) up to date, in essence, let's walk before we run 😄

Reply:
Yes, let’s tackle JS only in this PR. I would also like us to do the rest lint & format work inside ‘@wordpress/packages’ first and consume through ‘@wordpress/scripts’.

Reply:
Yeah, well that makes sense.